blocks/library/heading/index.js

```diff
@@ -29,6 +29,8 @@ registerBlockType( 'core/heading', {
 
 	className: false,
 
+	supportAnchor: true,
```

Comment:
Having an object for supports sounds good to me (maybe should truly be called _settings_), but I think `className` should still be opt-out.